src/vidible-player.js

```diff
@@ -11,9 +11,14 @@
             var videoReadyTimeout = 5000;
             var queue = [];
 
-            function loadScriptByVideoId(videoId, callback) {
-                var vidibleScriptUrl = 'http://delivery.vidible.tv/jsonp/pid=55c8aae9e4b0ca68372fb553/vid=' +
-                    videoId + '/55af9dcae4b02944c03a2eee.js';
+            var kAutoplayVidiblePlayer = '55c8aae9e4b0ca68372fb553';
```

Comment:
We should make player ID's something a user can configure outside of the directive.
I used a static one as you can see, but there's a task for adding configuration to it.